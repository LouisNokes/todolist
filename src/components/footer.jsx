import React from 'react';

export const Footer = () => {
    return (
        <footer className="mt-auto py-3 text-center container">
            <p>
                &copy; QA Ltd 2019-
            </p>
        </footer>
    )
}
