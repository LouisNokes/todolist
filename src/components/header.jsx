import logo from '../logo.svg';
import React from 'react';

export const Header = () => {
    return (
        <header>
            <nav className='navbar navbar-expand-sm'>
                <a href='https://www.qa.com'
                    className="navbar-brand"
                    target="_blank"
                    rel="noopener noreferrer"></a>
                <img src={logo} alt='QA Ltd' style={{ width: '100px' }}></img>
                <a className="navbar-brand" href="/"></a>
                <h1>Todo App</h1></nav>
        </header>
    )
}
