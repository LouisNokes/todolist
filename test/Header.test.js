import React from 'react';
import { create } from 'react-test-renderer';
import { Header } from '../src/components/header';

test('Header matches snapshot', () => {
    const header = create(<Header />);
    expect(header.toJSON()).toMatchSnapshot();
});