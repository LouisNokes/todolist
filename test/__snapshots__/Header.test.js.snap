// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<header>
  <nav
    className="navbar navbar-expand-sm"
  >
    <a
      className="navbar-brand"
      href="https://www.qa.com"
      rel="noopener noreferrer"
      target="_blank"
    />
    <a
      className="navbar-brand"
      href="/"
    />
    <h1>
      Todo App
    </h1>
  </nav>
</header>
`;
